cmd/storagenode-updater/main.go

```diff
@@ -311,5 +368,23 @@ func fileExists(filename string) bool {
 }
 
 func main() {
-	_ = rootCmd.Execute()
+	process.Exec(rootCmd)
+}
+
+// TODO: improve logging; other commands use zap but due to an apparent
+// windows bug we're unable to use the existing process logging infrastructure.
+func openLog() (closeFunc func() error, err error) {
+	closeFunc = func() error { return nil }
+
+	if runCfg.Log != "" {
+		logFile, err := os.OpenFile(runCfg.Log, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+		if err != nil {
+			log.Printf("error opening log file: %s", err)
+			return closeFunc, err
+		}
+		log.Printf("writing all further log output to %s", runCfg.Log)
```

Comment:
nit: remove "all further" from the message - it's obvious.

Reply:
done